modules/core/server/views/layout.server.view.html

```diff
@@ -56,7 +56,7 @@
 
   <!--Embedding The User Object-->
   <script type="text/javascript">
-    var user = {{{ user }}};
+    var user = {{ json user }};
```

Comment:
Also, adding a test for this would be cool. You could just copy+modify this one I made last week: https://github.com/Trustroots/trustroots/blob/master/modules/core/tests/server/core.server.config.tests.js